sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -291,6 +291,8 @@ def GetHeaders(  # pylint: disable=too-many-statements,too-many-branches
 
     if client_id is not None:
         headers[http_constants.HttpHeaders.ClientId] = client_id
```

Comment:
this bit can probably be removed as well, but won't block on this - LGTM thanks Bryan!